elastic/kibana
# [TableListView] Fix pagination state

This PR fixes a regression in the `<TableListView />` component that was introduced in https://github.com/elastic/kibana/pull/132321. 

The pagination was not handled anymore by the `EuiInMemoryTable` as the `onChange` prop was added for the sorting. When an `onChange` handler is passed then the pagination state **must** be kept at the consumer level and provided to the Eui table. But this is not enforced in anyway. I've opened an issue in eui (https://github.com/elastic/eui/issues/5928) to improve the documentation and behaviour to prevent those regressions.

## How to test

* Install all the sample data
* Navigate to the Visualize library --> There should be 2 pages of visualizations
* Click on "Page 2" link. You should see the second page

Fixes https://github.com/elastic/kibana/issues/132858
